CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 5.9.1(YYYY-MM-DD)
+
+### Enhancements
+* None
+
+### Fixed
+* [ObjectServer] Reporting to many errors from the native layer resulted in a native crash with `local reference table overflow`. (Issue [#249](https://github.com/realm/realm-java-private/issues/249), since 5.9.0)
```

Comment:
```suggestion
* [ObjectServer] Reporting too many errors from the native layer resulted in a native crash with `local reference table overflow`. (Issue [#249](https://github.com/realm/realm-java-private/issues/249), since 5.9.0)
```